Makefile

```diff
@@ -18,4 +18,11 @@ yaml_files:= \
 	$(root_dir)/misra/deviation_permit_template.yml
 $(call ci, yamllint, $(yaml_files))
 
-ci: license pylint yamllint
+c_files:=$(root_dir)/cppcheck-types.c
+mk_files:=$(root_dir)/Makefile $(root_dir)/ci.mk
```

Comment:
The `ci_dir` variable is defined to be used in `ci.mk` . Normally `ci.mk` assumes de ci-repo is available in a `ci` directory in the `root_dir`. However, since we are already in the ci directory we have to override it so that `ci.mk` uses the repo's root as the ci directory.

Reply:
Got it.